runtimes/php/php-apps.md

```diff
@@ -333,14 +333,17 @@ how to enable them.
 
 If you have a request about modules, feel free to contact our support at <support@clever-cloud.com>.
 
+Extensions enabled by default, add `ENABLE_<extension_name>`: `false` in your environment variables to disable them:
```

Comment:
it's actually ENABLE_foo = true and DISABLE_foo = true
ENABLE_foo = false is a no-op